scripts/suite_objects.py

```diff
@@ -2369,6 +2416,19 @@ def write(self, outfile, host_arglist, indent, const_mod,
                 # end if
                 pointer_var_set.append([name,kind,dimstr,vtype])
             # end for
+            # Any arguments used in variable transforms before or after the
+            # Scheme call? If so, declare local copy for reuse in the Group cap.
+            for ivar in self.transform_locals:
+                lname = ivar.get_prop_value('local_name')
```

Comment:
This may be a silly question, but what happens if two schemes have the same local_name for a variable and only one of them needs unit conversions, the other doesn't?

Reply:
Not at all, it's the crux of the changes in this PR.
In the modifications to the Scheme call_string(), we pass in a list of local_names changes if there are any transforms associated with the dummy argument in the call list. 
I'm updating the var_compatibility_test to reflect this.